code/dqc/timediff.q

```diff
@@ -0,0 +1,13 @@
+\d .dqc
+
+/ Takes a table name tn and two column names ca and cb, as well as a percentage
+/ pt and a tolerance tl as a timespan (eg 0D00:00:00.001).
+timediff:{[tn;ca;cb;pt;tl]
+  .lg.o[`timediff;"Checking the time differences in columns ",(", "sv string(ca;cb))," of table ",(string tn)];
+  ot:$[tl>re:1-(sum a)%count a:tl>(tn ca)-tn cb;
```

Comment:
is this comparison correct? if I'm reading it right, you're comparing the tolerance timespace to the percentage of times that differ by greater than the tolerance. Should this be

```suggestion
  ot:$[pt>re:1-(sum a)%count a:tl>(tn ca)-tn cb;
```

?